app/admin/projects.rb

```diff
@@ -0,0 +1,84 @@
+ActiveAdmin.register Project do
+  permit_params :name, :short_description, :long_description, :target_amount, :category_id, :thumbnail, :landscape
+
+  index do
+    selectable_column
+    column :name
+    column :target_amount
+    column :created_at
+    actions
+  end
+
+  filter :name
+  filter :target_amount
+  filter :created_at
+
+  show do |project|
+    columns do
+      column do
+        span t('.percentage_of_completion', percentage: project.decorate.percentage_of_completion)
```

Comment:
En règle générale : ne décore pas dans la vue. Ça peut faire des trucs dégueux en terme de performance. Le controller doit retourner les objets prêts à être consommés par la vue. 

Le readme de Draper a toute une section là dessus : https://github.com/drapergem/draper#when-to-decorate-objects

Dans le cas d'active admin c'est un peu chiant de modifier le controller mais il existe une méthode toute faite pour utiliser un décorateur. Je te laisse chercher tu devrais trouver très facilement.

Reply:
Yes j'en étais arrivé au même constat mais je ne voulais pas modifier le controller à nouveau.
Je ne savais pour la méthode toute faite je vais regarder ça alors.